common/static/common/js/discussion/discussion_router.js

```diff
@@ -89,16 +100,44 @@
                 if (this.thread) {
                     return this.nav.setActiveThread(this.thread.get("id"));
                 } else {
-                    return this.nav.goHome;
+                    return this.nav.goHome();
                 }
             };
 
             DiscussionRouter.prototype.showThread = function(forum_name, thread_id) {
+                var self;
+                self = this;
```

Comment:
@bjacobel good point. To defend myself, this is cherry-picked from fix that was made in coffee, and this is how it coffee gernerates code :)